src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandArgs.cs

```diff
@@ -159,6 +160,10 @@ private void ValidateUploadOptions()
         {
             throw new OctoshiftCliException("The AWS S3 bucket name must be provided with --aws-bucket-name if other AWS S3 upload options are set.");
         }
+        if (UseGithubStorage && AwsBucketName.HasValue())
+        {
+            throw new OctoshiftCliException("The --use-github-storage flag was provided with an AWS S3 Bucket name. Archive cannot be uploaded to both locations.");
+        }
```

Comment:
I guess we also need a check for when `--use-github-storage` is provided with `AzureStorageConnectionString`

Reply:
Updated and addressed the comments with [this]([420db29](https://github.com/github/gh-gei/pull/1270/commits/420db29ceb3debcd1d175f0623d6c1a214961d0a)) commit 👍🏽